src/main/scala/net/ceedubs/ficus/readers/ArbitraryTypeReader.scala

```diff
@@ -8,7 +8,45 @@ import macrocompat.bundle
 import scala.reflect.macros.blackbox
 
 trait ArbitraryTypeReader {
-  implicit def arbitraryTypeValueReader[T]: ValueReader[T] = macro ArbitraryTypeReaderMacros.arbitraryTypeValueReader[T]
+  implicit def arbitraryTypeValueReader[T]: ValueReader[T] =
+    macro ArbitraryTypeReaderMacros.arbitraryTypeValueReader[T]
+}
+
+/**
+  * Helper object to get the current name mapper
+  */
+object NameMapper {
+
+  /**
+    * Gets the name mapper from the implicit scope
+    * @param nameMapper The name mapper from the implicit scope, or the default name mapper if not found
+    * @return The name mapper to be used in current implicit scope
+    */
+  def get(implicit nameMapper: NameMapper = DefaultNameMapper) = nameMapper
```

Comment:
hmm, not sure how I feel about the name `get` here, usually we made such implicit instance summon helper method the `apply` method.